plugins/editors/tinymce/tinymce.php

```diff
@@ -779,7 +779,7 @@ protected static function getGlobalFilters()
 			}
 			else
 			{
-				// Blacklist or whitelist.
+				// DisallowList or AllowList.
```

Comment:
```suggestion
				// Disallow List or Allow List.
```
Formalise the wording to show as separate words.